tests/plugins/wysiwygarea/detached.js

```diff
@@ -0,0 +1,260 @@
+/* bender-tags: editor, 4462 */
+/* bender-ckeditor-plugins: basicstyles,toolbar */
+
+( function() {
+	'use strict';
+
+	function isSupportedEnvironment() {
+		// Ignore all IE versions below 11.
+		return !( CKEDITOR.env.ie && !CKEDITOR.env.edge && CKEDITOR.env.version < 11 );
+	}
+
+	bender.test( {
+		'test reattached editor contains the same data with observed default dom object': function() {
+			if ( !isSupportedEnvironment() ) {
+				assert.ignore();
+			}
+
+			var startupData = '<p>CKEditor4</p>';
```

Comment:
AFAICS this data is the same for each test, so we could define it before and reuse it.